test/basic-querying.test.js

```diff
@@ -153,6 +153,26 @@ describe('basic-querying', function() {
           done();
         });
     });
+    it('should query by ids and null condition', function(done) {
+      // due to some connectors not being assigning null value by default
+      // the query is filtered by the fields which may consist null value
+      // or not exist at all
+      User.findByIds([
+        createdUsers[0].id,
+        createdUsers[1].id],
+        {fields: {name: true, vip: true}}, function(err, users) {
```

Comment:
@paulussup Good question. Your test case was not wrong however it had a loophole where it would not check for connectors which does not put `null` values for fields which are not set (it would instead have it as undefined, for example mongodb).

Improved the test case to take care of that situation by filtering by fields (name and vip) and then get those instances back and then perform a check if they are either `null` or `undefined`. After fixing this it works with the downstream connectors such as mongodb and cassandra.